scripts/gh_pr.sh

```
@@ -0,0 +1,161 @@
+        exit 1
+    fi
+
+    local current_branch=$(git rev-parse --abbrev-ref HEAD)
+    echo -e "${BOLD}Merging PR #$pr_id into $current_branch...${NC}"
+
+    if ! git diff-index --quiet HEAD --; then
+        echo -e "${RED}Error: You have unstaged changes. Please commit or stash them first.${NC}"
+        exit 1
+    fi
+
+    git fetch origin pull/$pr_id/head || { echo -e "${RED}Error: Failed to fetch PR.${NC}"; exit 1; }
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)